tests/lib/store.sh

```diff
@@ -44,6 +44,47 @@ make_snap_installable(){
     new_snap_revision "$dir" "$snap_path"
 }
 
+make_snap_installable_with_id(){
+    local dir="$1"
+    local snap_path="$2"
+    local snap_id="$3"
+
+    if ! command -v yaml2json; then
+        snap install remarshal
+    fi
+    if ! command -v jq; then
```

Comment:
the problem with this is that when we are not in core16 it forces to download the core snap. I should be nice if we could have a function which checks which jq to download based on the spread system.
This is not a blocker, just an improvement

Reply:
ok, thanks for the review, it's just that having these tools available is really convenient for writing test helpers like this and they are available as snaps and it's really easy to install as a snap :smile: